ppr-ui/src/views/mhrSearch/MHRSearch.vue

```diff
@@ -4,17 +4,21 @@
       <v-progress-circular color="primary" size="50" indeterminate />
     </v-overlay>
     <v-container class="container">
-      <b :class="$style['search-title']">Selection List</b>
+      <div style="display: flex;">
+        <v-icon color="#212529" size="32" class="pr-1">mdi-home</v-icon>
```

Comment:
Inline styling and hex should ideally be kept out of the template. 
Use local classes or base classes where possible.

ie, lets say the them colour $gray7 was changed in the theme, this would not as an example.

Reply:
Agree, we shouldn't be using inline styles. Quick but dirty way. Will update.